### PR TITLE
Prettify updates

### PR DESCRIPTION
**Pull request by jonasdt**
See commit messages. 

The purpose of eb61451 and 915a1ad is to make the code before and after prettify look the same, the only difference is the syntax highlighting.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
this doesn't look right?

**Comment by GrahamCampbell** [comment]
you've deleted all the async code

**Comment by jonasdt** [comment]
Yup, the file is included in the response now. I don't see a performance difference, and now it works when I'm on the plane :)

File: src/Whoops/Resources/views/frame_code.html.php [hunk 1]
```diff
@@ -29,20 +29,6 @@
             $start = key($range) + 1;
             $code  = join("\n", $range);
         ?>
-            <style>
-             #frame-code-0 .linenums li:nth-child(<?= $line-$start ?>) {
-                 background-color: rgba(255, 100, 100, .07);
```
Comment (denis-sokolov) [inline src/Whoops/Resources/views/frame_code.html.php]: You probably did not mean to also remove background-color highlight from the lines.

 - [ ] Clarify the background-color highlight.

**Comment by denis-sokolov** [comment]
I agree with removing reliance on 3rd party servers.

File: src/Whoops/Resources/views/frame_code.html.php (hunk 1 above)
Comment (jonasdt) [inline src/Whoops/Resources/views/frame_code.html.php]: The background-color is defined in [whoops.base.css](https://github.com/jonasdt/whoops/blob/915a1ad1762e443280df6f42e35033a0aaa3bea1/src/Whoops/Resources/css/whoops.base.css#L353-L358).

**Comment by masterwto** [comment]
I agree with removing reliance on 3rd party servers, too, as my project is developed and deployed off internet.

**Comment by masterwto** [comment]
Has been talking for almost a whole year!
I think it's time to merge this PR.

**Comment by denis-sokolov** [comment]
Merged in 1414942. Took a bit of work to get this up-to-date, but I kept your authorship, @jonasdt. Thanks for your contribution!

**Pull request closed without merging**